server/lib/tuist_web/controllers/apple_app_site_association_controller.ex

```diff
@@ -0,0 +1,41 @@
+defmodule TuistWeb.AppleAppSiteAssociationController do
+  use TuistWeb, :controller
+
+  alias Tuist.Environment
+
+  @doc """
+  Serves the Apple App Site Association file dynamically based on the environment.
+  """
+  def show(conn, _params) do
+    app_id = get_app_id()
+
+    association = %{
+      applinks: %{
+        apps: [],
+        details: [
+          %{
+            appID: app_id,
+            paths: ["/*/*/previews/*"]
+          }
+        ]
+      }
+    }
+
+    conn
+    |> put_resp_content_type("application/json")
+    |> json(association)
```

Comment:
Isn't `json()` already setting the content type of the response?

Reply:
oh yeah, you're right.